packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -176,18 +168,6 @@ export const createPortal = enableNewReconciler
 export const createComponentSelector = enableNewReconciler
   ? createComponentSelector_new
   : createComponentSelector_old;
-export const DefaultEventPriority = enableNewReconciler
-  ? DefaultEventPriority_new
-  : DefaultEventPriority_old;
-export const DiscreteEventPriority = enableNewReconciler
-  ? DiscreteEventPriority_new
-  : DiscreteEventPriority_old;
-export const ContinuousEventPriority = enableNewReconciler
-  ? ContinuousEventPriority_new
-  : ContinuousEventPriority_old;
-export const IdleEventPriority = enableNewReconciler
-  ? IdleEventPriority_new
-  : IdleEventPriority_old;
```

Comment:
We currently reference these values in README. If we're removing them, can you update the README with a recommendation of what to do? E.g. just hardcoding numbers or telling people to look at this particular file. https://github.com/facebook/react/blob/master/packages/react-reconciler/README.md#getcurrenteventpriority